packages/Discretization/src/DTK_PointSearch_def.hpp

```diff
@@ -761,11 +850,11 @@ PointSearch<DeviceType>::computeNCellsPerTopology(
         n_cells_per_topo[dtk_cell_topo] += 1;
     }
 
-    // We do not support meshes that contains both 2D and 3D cells. All the
-    // cells are either 2D or 3D
     Topologies topologies;
     _dim = topologies[dtk_cell_topo].dim;
 #if HAVE_DTK_DBC
+    // We do not support meshes that contains both 2D and 3D cells. All the
+    // cells are either 2D or 3D
```

Comment:
Also what changed here, I cannot tell.

Reply:
I just move the comment so that the comment is inside below the `#if` like it is for the others